### PR TITLE
Checklists

### PR DESCRIPTION
**Pull request by nelsonic**
My goal with this pull request is to: 
+ [x] Clarify the "What?" section of the readme for https://github.com/dwyl/product-roadmap/issues/9
+ [x] Give some detail as to the MVP we are building.

### CODE REVIEW

**Comment by nelsonic** [comment]
This is the Post-it Prototype of my "Morning Routine" checklist that I follow each morning:
nelson-mourning-routine-checklist-postit-prototype

Reduced image dimensions and Optimised:
nelson-mourning-routine-checklist-postit-prototype-min
Slight loss of quality on text but 95% reduction in file size!

**Comment by nelsonic** [comment]
@iteles please review & merge when you can.
Thanks. 🌻

File: checklists.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+# _What?_
+
+Our mission is simple:
+build the simplest possible UI/UX
+for creating, sharing and performing repeatable checklists.
+
+And in doing this we will _systematically_:
+1. Build a tool that we will use ourselves to:
+  1. maximise our _personal_ effectiveness
+  2.
```
Comment (SimonLab) [inline checklists.md]: I don't think the indentation is correct on this list:
image

File: checklists.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+A checklist is the most effective way of
+keeping track of the list of things to be done
+and ensuring that nothing is missed.
+
+https://www.google.com/search?q=checklist
+![image](https://user-images.githubusercontent.com/194400/59164503-85f47a80-8b05-11e9-9799-26446f8aff0c.png)
+> "_a list of items required, things to be done,
+or points to be considered, used as a reminder._"
+
+## The Checklist Manifesto
```
Comment (SimonLab) [inline checklists.md]: Looks like this subtitle is not rendered properly:
image

File: checklists.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+
+![checklist-manifesto-best-seller-categories](https://user-images.githubusercontent.com/194400/56946615-81aa6a00-6b22-11e9-923e-9eb8ce9171f6.png)
+
+> <sup>1</sup>_**Note**: the book was published in 2009 (as a hardback) but the **paperback** of 2011 is the one that has sold
+the numbers_.
+
+
+# _Why?_
+
+## Why does this _matter_?
```
Comment (SimonLab) [inline checklists.md]: Subtitle rendering issue:
image

File: checklists.md (hunk 1 above)
Comment (nelsonic) [reply checklists.md]: @SimonLab good spot. I've added the missing bullet point.

File: checklists.md (hunk 3 above)
Comment (nelsonic) [reply checklists.md]: @SimonLab thanks for taking the time to read the PR and make these comments.
I think I've resolved the Markdown issues.

**Comment by nelsonic** [comment]
@iteles I've made the amendments suggested by @SimonLab (_12 days ago_). 📝 
Please review and merge when you can. 👀 
Thanks. ☀️

**Review by iteles — APPROVED** [review]
Oops. My apologies, I don't know how I managed to miss this 😞

**Comment by nelsonic** [comment]
@iteles thanks for merging. 👍

**Pull request merged**